src/main/resources/application.yml

```
@@ -1,14 +1,45 @@
+discodeit:
+  storage:
+    type: local
+    local:
+      root-path: .discodeit/storage
+
 spring:
-  application:
-    name: discodeit
+  jpa:
+    open-in-view: false
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+    properties:
+      hibernate:
+        default_batch_fetch_size: 100
+    sql:
+      init:
+        mode: always
+        schema-locations: classpath:schema.sql
```

```suggestion
  sql:
    init:
      mode: always
      schema-locations: classpath:schema.sql
```
들여쓰기가 한칸 더 되어 있네요.
추가로, schema-locations의 기본값이 schema.sql이기 때문에 명시할 필요는 없습니다.

그리고 ddl-auto 기능과 sql init 기능 중에 하나만 사용하시길 추천 드립니다.
ddl-auto가 실행된 다음에 sql init이 실행되기 때문에 쿼리 오류가 발생할 가능성이 높습니다.